screens/login.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
-import { StyleSheet, Text, View} from 'react-native';
+import { StyleSheet, Text, View, Dimensions} from 'react-native';
 import { Font } from 'expo';
 
+const window = Dimensions.get("window")
 
 export default class Login extends React.Component {
   state = {
     fontLoaded: false,
   };
   async componentDidMount() {
     await Font.loadAsync({
-      'Montserrat-Regular': require('/Users/nona/shopu/assets/Montserrat-Regular.otf'),
-      'Montserrat-Medium': require('/Users/nona/shopu/assets/Montserrat-Medium.otf'),
+      'Montserrat-Regular': require('/Users/nona/shopu/assets/fonts/Montserrat-Regular.otf'),
```

Comment:
Routes to any files or assets in a project should not start with your local computer's home file. The moment I install this app on my phone, the app will look for the font with that path which won't exist.

Instead you should start your path from the current file that you're at - it should look something like `./../assets/fonts/Montserrat-Regular.otf`